src/Mappers/UpperCaseMapper.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We do not add strict types to Spatie packages

Reply:
👌